vsintegration/src/FSharp.Editor/LanguageService/LanguageService.fs

```diff
@@ -89,24 +95,28 @@ type Refreshable<'T> = 'T * (bool -> 'T)
 // The main entrypoints are TryGetOptionsForDocumentOrProject and TryGetOptionsForEditingDocumentOrProject.
 
 
-[<Export(typeof<FSharpProjectOptionsManager>); Composition.Shared>]
-type internal FSharpProjectOptionsManager 
+[<Export(typeof<ProjectInfoManager>); Composition.Shared>]
+type internal ProjectInfoManager 
```

Comment:
You might want to undo this renaming, to make the PR really minimal.

Reply:
(Looking further - yes, please undo the renaming for this PR so we can check it carefully, thanks)

Reply:
@dsyme ... I'm good with the rename.  It's back to what it originally was :-)

Reply:
I'm fine with the rename, I just wanted the diff minimized (i.e. do it in a separate PR)

Reply:
I know I'm sounding like a broken record, but the diff would be much smaller without this rename.  LanguageService.fs is a PITA and we need to really scrutinize and minimize all changes to it, and potentially undo them if we get them wrong.